go-controller/pkg/ovn/loadbalancer.go

```diff
@@ -214,7 +139,7 @@ func (ovn *Controller) createLoadBalancerRejectACL(lb, sourceIP string, sourcePo
 	}
 	vip := util.JoinHostPortInt32(sourceIP, sourcePort)
 	// NOTE: doesn't use vip, to avoid having brackets in the name with IPv6
-	aclName := ovn.generateACLNameForOVNCommand(lb, sourceIP, sourcePort)
+	aclName := loadbalancer.GenerateACLNameForOVNCommand(lb, sourceIP, sourcePort)
 	// If ovn-k8s was restarted, we lost the cache, and an ACL may already exist in OVN. In that case we need to check
 	// using ACL name
 	aclUUID, stderr, err := util.RunOVNNbctl("--data=bare", "--no-heading", "--columns=_uuid", "find", "acl",
```

Comment:
it seems like the code here could be rewritten to use `loadbalancer.GetACLByName` (in which case maybe `loadbalancer.GenerateACLNameForOVNCommand` should be moved into that too)

Reply:
indeed, actually ACLs should go to a different package, let me split it out from loadbalancers